image_width: 1280
image_height: 720
camera_name: camera
camera_matrix:
  rows: 3
  cols: 3
  data: [386.522061743, 0.000000, 650.538158756, 0.000000, 386.464342885, 358.102279975, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.039648906, -0.006374964598, 0.00267253, -0.001376483, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [386.522061743, 0.000000, 650.538158756, 0.000000, 0.000000, 386.464342885, 358.102279975, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]